recipes/pp-sketchlib/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "pp-sketchlib" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/johnlees/{{ name|lower }}/archive/v{{ version }}.tar.gz
+  sha256: cc78e11f7afef7128fed4b048ccdf44e968825fde257d5b25925f9fb68a7c32a
+  patches:
+    - 0001-Replace-cmake-with-make.patch  # [linux64 and cuda_compiler_version != "None"]
+    - 0002-Pass-libloc-to-make-command.patch  # [linux64 and cuda_compiler_version != "None"]
+    - 0003-Fix-target-dependencies-in-Makefile.patch  # [linux64 and cuda_compiler_version != "None"]
+
+build:
+  number: 0
+  skip: True  # [py2k or win]
+  script:
+    - "{{ PYTHON }} -m pip install . --no-deps -vv"
+  missing_dso_whitelist:
+    - "*/libcuda.*"
+  entry_points:
+    - poppunk_sketch = pp_sketch.__main__:main
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}  # [linux64]
+    - cmake
+  host:
+    - python
+    - pip
+    - pybind11
+    - highfive
+    - zlib
+    - eigen
+    - armadillo
+    - hdf5
+    - openblas
+  run:
+    - python
+    - numpy
+    - zlib
+    - hdf5
+    - h5py
+    - openblas
+    - libgfortran-ng
+    - libgfortran  # [osx]
+
+test:
+  imports:
+    - pp_sketch
+  commands:
+    - poppunk_sketch --version
+
+about:
+   home: https://github.com/johnlees/pp-sketchlib
+   license: Apache-2.0
+   license_family: APACHE
+   license_file:
+    - NOTICE
+    - LICENSE
+    - LICENSE_bindash
+    - LICENSE_nthash
```

Comment:
Are all of these apche? Otherwise we should use an SPDX license expression.

Reply:
Apache 2.0 and MIT. I have now added an appropriate SPDX expression as suggested